lib/matplotlib/_docstring.pyi

```diff
@@ -4,6 +4,9 @@ from typing import Any, Callable, TypeVar, overload
 _T = TypeVar('_T')
 
 
+def _kwarg_doc(text: str) -> Callable[[_T], _T]: ...
```

Comment:
```suggestion
def kwarg_doc(text: str) -> Callable[[_T], _T]: ...
```

Not sure why stubtest didn't flag this, when I think it _did_ flag it as missing previously...

Reply:
Oh... I ignore all errors from private modules... which maybe we want to be a bit more strict there, but that is outside of the scope here (Quick check shows ~100 "errors" identified in such modules, many of which are C-implemented functions not being represented, etc.)